src/Hosting/Hosting/src/StaticWebAssets/StaticWebAssetsLoader.cs

```diff
@@ -93,12 +65,11 @@ internal static (Stream, bool) ResolveManifest(IWebHostEnvironment environment,
             }
         }
 
-        private static IEnumerable<(string candidatePath, bool isJson)> ResolveRelativeToAssembly(IWebHostEnvironment environment)
+        private static string ResolveRelativeToAssembly(IWebHostEnvironment environment)
         {
             var assembly = Assembly.Load(environment.ApplicationName);
             var basePath = string.IsNullOrEmpty(assembly.Location) ? AppContext.BaseDirectory : Path.GetDirectoryName(assembly.Location);
-            yield return (Path.Combine(basePath!, $"{environment.ApplicationName}.staticwebassets.runtime.json"), isJson: true);
-            yield return (Path.Combine(basePath!, $"{environment.ApplicationName}.StaticWebAssets.xml"), isJson: false);
+            return Path.Combine(basePath!, $"{environment.ApplicationName}.staticwebassets.runtime.json");
```

Comment:
I don't have context on the changes in this file so I'll just assume it's all great 👍

Reply:
Yeah, this is just cleanup from preview7. Since Blazor Desktop and ASP.NET Core consume the same manifest now I did the cleanup in one go to avoid having to update stuff that I would have to delete.